numpy/core/src/multiarray/dtype_transfer.c

```diff
@@ -488,7 +457,8 @@ wrap_aligned_contig_transfer_function(
             PyArray_StridedUnaryOp *wrapped, NpyAuxData *wrappeddata,
             int init_dest,
             PyArray_StridedUnaryOp **out_stransfer,
-            NpyAuxData **out_transferdata)
+            NpyAuxData **out_transferdata,
+            int out_needs_api)
```

Comment:
```suggestion
            npy_bool out_needs_api)
```

Reply:
How far would we want this change to bubble up the call stack?

Reply:
Let's just declare it out of scope for now.

Reply:
Opened gh-15814